cmd/influxd/run/server.go

```diff
@@ -487,6 +487,16 @@ func (s *Server) Open() error {
 		// so it only logs as is appropriate.
 		s.QueryExecutor.TaskManager.Logger = s.Logger
 	}
+	if s.config.Data.QueryLogPath != "" {
+		path := s.config.Data.QueryLogPath
+		flw := query.NewFileLogWatcher(s.QueryExecutor, path, s.Logger, s.config.Logging.Format)
+		if flw != nil {
+			s.QueryExecutor.WithLogWriter(flw, context.Background())
+		} else {
+			s.Logger.Error("error creating log writer", zap.String("path", path))
```

Comment:
Yeah, if `NewFileLogWatcher` returned an error, you could log the error right here instead of this generic log message.